tests/queries/0_stateless/02000_join_on_const.sql

```diff
@@ -43,6 +43,8 @@ SELECT * FROM t1 RIGHT JOIN t2 ON NULL ORDER BY t1.id NULLS FIRST, t2.id SETTING
 SELECT '- full -';
 SELECT * FROM t1 FULL JOIN t2 ON NULL ORDER BY t1.id NULLS FIRST, t2.id SETTINGS join_use_nulls = 1;
 
```

Comment:
It is not clear for me why we need to explicitly disabled it here, then enable it later. Could you please explain?